.travis.yml

```diff
@@ -1,57 +1,9 @@
-language: php
+version: ~> 1.0
```

Comment:
Could you replace this with https://github.com/silverstripe/gha-ci - we're replacing travis with this

You may need to do this in a separate PR that we merge before this one

Reply:
I've raised #140

Can this happen after this PR?

Reply:
yeah after is fine